src/app/zeko/da-layer/da_utils/src/DataAvailability.json

```diff
@@ -0,0 +1 @@
+../../artifacts/contracts/DataAvailability.sol/DataAvailability.json
```

Comment:
This is a broken symlink? What is it supposed to point to?

Reply:
The compilation of solidity code generates this artifact and rust than creates structs from this artifact.

Reply:
You should not do this and instead use an environment variable or similar.